backend/services/chat_service.py

```
@@ -40,11 +41,19 @@ def _get_history(session_id: str) -> ChatMessageHistory:
     if history is None:
-        db = next(get_db())
-        history = chat_history_crud.get_chat_history(db, session_id)
-        if history is None:
+        with get_db() as db:
+            loaded_history = chat_history_crud.get_chat_history(db, session_id)
+        if loaded_history and getattr(loaded_history, "context", None):
+            try:
+                msgs = messages_from_dict(json.loads(loaded_history.context))
+                history = ChatMessageHistory()
+                history.messages = msgs
+            except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)